ros2_socketcan_msgs/msg/Frame.msg

```diff
@@ -0,0 +1,8 @@
+std_msgs/Header header
```

Comment:
The maintainer, @mathias-luedtke, appears to have abandoned the `ros_canopen` repository. His last reply on issue 375 was from January of 2020. There are 59 open issues and the last time he replied to one was January of 2021 (22 months ago). His [contribution history](https://github.com/mathias-luedtke?tab=overview&from=2022-11-01&to=2022-11-02) shows that he has only barely touched Github in the last year. The most recent update to any branch of this repository was 11 months ago. This repo is dead and others have realized this and started making their own versions like https://github.com/Factor-Robotics/odrive_ros2_control but this only handles the need for implementing the CANOpen portions of the repository and not a basic CAN reader/writer. The `ros2_socketcan` repo has, from what I've seen, become the de-facto interface for using SocketCAN with ROS2.

Reply:
Have you seen https://github.com/ros-industrial/ros2_canopen? This where the active development is happening.
@ipa-cmh: FYI

Reply:
@mathias-luedtke Thank you for the reference. BTW, no ill will intended with the information above was just trying to prove a point (we all have things change and no one person can maintain a thing forever). Also, it looks like it is still the case that the new repository you mentioned is focused strictly on CANOpen and does not have the same SocketCAN utilities that the ROS1 version did. Please correct me if I'm wrong.

Reply:
@JWhitleyWork as you have noticed, the main work in 2022 is focussed on https://github.com/ros-industrial/ros2_canopen now. But as you pointed out, there are no socketcan utitilies in ros2_canopen as this is handled by the integrated lely canopen libraries. We made this decision mainly because ros2_socketcan was already there when we started development.